website/_redirects

```diff
@@ -45,9 +37,13 @@
 /docs/internals/security                             /docs/security                                    301!
 /docs/acl/                                           /docs/security/acl/                               301!
 /docs/acl/acl-system                                 /docs/security/acl/acl-system                     301!
-/docs/acl/acl-rules                                  /docs/security/acl/acl-rules                       301!
+/docs/agent/acl-system.html                          /docs/security/acl/acl-system                     301!
```

Comment:
```suggestion
```
Suggested adding this one back grouped with the other `/docs/agent` redirects in a prior comment.